templates/saphana.tpl.yaml

```diff
@@ -0,0 +1,179 @@
+    template.kubevirt.io/version: "{{ lookup('env', 'VERSION') | default('devel', true) }}"
+{% if item.default %}
+    template.kubevirt.io/default-os-variant: "true"
+{% endif %}
+objects:
+  - apiVersion: kubevirt.io/v1
+    kind: VirtualMachine
+    metadata:
+      labels:
+        vm.kubevirt.io/template: {{ os }}-{{ item.workload }}-{{ item.flavor }}
+        vm.kubevirt.io/template.version: "{{ lookup('env', 'VERSION') | default('devel', true) }}"
+        vm.kubevirt.io/template.revision: "{{ lookup('env', 'REVISION') | default(1, true) }}"
+        app: ${NAME}
+      name: ${NAME}
+    spec:
+      dataVolumeTemplates:
+        - apiVersion: cdi.kubevirt.io/v1beta1
+          kind: DataVolume
+          metadata:
+            name: ${NAME}
+          spec:
+            storage:
+              resources:
+                requests:
+                  storage: 50Gi
+            source:
+              registry:
+                url: ${SRC_CONTAINERDISK}
+                pullMethod: node
+      running: false
+      template:
+        metadata:
+          annotations:
+            vm.kubevirt.io/os: "{{ os }}"
+            vm.kubevirt.io/workload: "{{ item.workload }}"
+            vm.kubevirt.io/flavor: "{{ item.flavor }}"
+          labels:
+            kubevirt.io/domain: ${NAME}
+            kubevirt.io/size: {{ item.flavor }}
+        spec:
+          hostname: ${NAME}
+          domain:
+            ioThreadsPolicy: auto
+            cpu:
+              cores: {{ '${{CPU_CORES}}' }}
+              threads: {{ '${{CPU_THREADS}}' }}
+              sockets: {{ '${{CPU_SOCKETS}}' }}
+              model: host-passthrough
+              isolateEmulatorThread: true
+              dedicatedCpuPlacement: true
+              features:
+                - name: "invtsc"
+                  policy: "require"
+              numa:
+                guestMappingPassthrough : {}
+            devices:
+              blockMultiQueue: true
+              networkInterfaceMultiqueue: true
+              disks:
+                - disk:
+                    bus: virtio
+                  dedicatedIOThread: true
+                  name: ${NAME}
+                - disk:
+                    bus: virtio
+                  name: cloudinitdisk
+                - disk:
+                    bus: virtio
+                  name: downwardmetrics
+              interfaces:
+                - masquerade: {}
+                  name: default
+                - name: sriov-net1
+                  sriov: {}
+                - name: sriov-net2
+                  sriov: {}
+                - name: sriov-net3
+                  sriov: {}
+            machine:
+              type: ""
+            resources:
+              requests:
+                memory: ${MEMORY_OVERHEAD}
+            memory:
+              guest: ${MEMORY}
+              hugepages:
+                pageSize: ${HUGEPAGES_PAGE_SIZE}
+          terminationGracePeriodSeconds: 0
```

Comment:
According to @rnetser terminationGracePeriodSeconds can't be zero

Reply:
Thanks @rnetser that is a beginner mistake :)

Reply:
Fixed.